gdi/opentelemetry/components/fluentd-receiver.rst

```diff
@@ -11,7 +11,7 @@ The Fluent Forward receiver allows the Splunk Distribution of OpenTelemetry Coll
 
 The receiver accepts data formatted as Fluent Forward events through a TCP connection. All three Fluent event types, message, forward, and packed forward, are supported, including compressed packed forward.
 
-.. note:: Fluentd is bundled in the Splunk Distribution of OpenTelemetry Collector.
```

Comment:
this is important info imo and these changes should be additive (included fluentd/td-agent)

Reply:
Let me recover that.